Arduino/Propulsion/src/PropulsionModule.cpp

```diff
@@ -54,7 +70,7 @@ int16_t byteToInt(int16_t index);
 void setup() {
   Serial.begin(SERIAL_BAUD);
   Serial.setTimeout(serialTimeout);
-  qik.init();
+  initMotor();
```

Comment:
What was your reasoning for moving away from a call to the library directly?
Further, initMotor is a poor name as you aren't initializing a motor, you're initializing the driver board itself, which then 'arms' two motors. At the same time, I don't like something like initController, as that is ambiguous as to whether it means microcontroller or motor controller. Maybe something like initDriver?

Reply:
I'm open to either renaming (e.g. `initDriver` or `initController`) or removing this indirection. What makes more sense to you?

Reply:
Let's do initDriver

Reply:
Alrighty